src/scripts/cfo.zig

```diff
@@ -267,6 +274,20 @@ fn run_fuzzers(
             }
         }
 
+        // Increment the runtime of running tasks.
+        for (children) |*fuzzer_or_null| {
+            if (fuzzer_or_null.*) |*fuzzer| {
+                if (tasks.map.get(.{
+                    .fuzzer = std.meta.stringToEnum(Fuzzer, fuzzer.seed.fuzzer).?,
+                    .commit = fuzzer.seed.commit_sha,
+                })) |task_index| {
+                    tasks.runtimes[task_index] += 1;
+                } else {
+                    // This is a leftover fuzzer from a now-cancelled task.
```

Comment:
An alternative would be to make Tasks append-only, and mark inactive tasks (or, rather, track the generation number, and only consider tasks from the latest generation to be active)